src/components/Trade/hooks/useTradeAmounts.tsx

```diff
@@ -162,11 +152,10 @@ export const useTradeAmounts = () => {
 
       if (!bestTradeSwapper) return
       const state = store.getState()
-      const sellAssetAccountId = getFirstAccountIdFromChainId(sellAsset.chainId)
       const sellAssetAccountIds = selectPortfolioAccountIdsByAssetId(state, {
```

Comment:
`selectPortfolioAccountsGroupedByNumberByChainId`?

Reply:
@gomesalexandre I'll leave this as is so we can get this PR in.